spec/services/projects/copy_service_integration_spec.rb

```diff
@@ -209,6 +210,69 @@
           end
         end
 
+        context "with calculated custom fields", with_flag: { calculated_value_project_attribute: true } do
+          let(:integer_custom_field) { create(:integer_project_custom_field, projects: [source]) }
+          let(:calculated_custom_field) do
+            create(:calculated_value_project_custom_field, :skip_validations,
+                   projects: [source],
+                   formula: "{{cf_#{integer_custom_field.id}}} * 2")
+          end
+
+          before do
+            source.custom_field_values = { integer_custom_field.id => 5 }
+            source.save!
+            source.calculate_custom_fields([calculated_custom_field])
+            source.save!
+          end
+
+          it "copies the custom fields and recalculates values" do
```

Comment:
There should be a test to check that changing custom field values in copy form leads to them being recalculated